modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vmdk-suma.adoc

```diff
@@ -0,0 +1,320 @@
+The directory structure must look as follows:
+
+----
+<root directory>
+└── meta-data
+└── network-config
+└── user-data
+----
+
+If you intend to configure a QEMU/KVM virtual machine, provide the path to the [path]``config.ign`` as an attribute of the qemu command.
+For example:
+
+----
+-fw_cfg name=opt/com.coreos/config,file=PATH_TO_config.ign
+----
+
+The [systemitem]``Cloud Init`` allows many management options. For a complete specification, refer to Cloud Init specification (https://cloudinit.readthedocs.io/en/latest/index.html).
+
+// FIXME: we probably do not need this section
+=== Set Up Root Password Using Cloud Init
+
+You need to hash the root password, for example, by using the [command]``openssl`` command:
+
+----
+openssl passwd -6
+----
+
+The command creates a hash of the password you chose.
+Use this hash as the value of the [literal]``password`` attribute.
+
+Prepare the needed configuration files using the following commands:
+
+----
+touch network-config
+touch meta-data
+----
+
+Create a file named [path]``user-data`` with the following content:
+
+----
+#cloud-config
+chpasswd:
+  expire: false
+  users:
+    - name: root
+      password: $2a$10$qV298UV11u9lCFDjpHpCUe1cErBiVR.G3shukxs3.2PAO1xhJWs0K
+----
+
+Prepare [systemitem]``Cloud Init`` ISO file using the command:
+
+----
+mkisofs -rational-rock -joliet -o suma_cloudinit.iso -V cidata network-config meta-data user-data
+----
+
+Attach the created [path]``suma_cloudinit.iso`` file as a volume to the creating virtual machine.
+This particular example is setting [systemitem]``root`` password to `linux`.
+Substitute your password hash for the one in this example
+
+
+
+
+[[quickstart.sect.vmware.settings]]
+== {productname} Virtual Machine Settings - VMware
+
+This sections describes VMware configurations, focusing on the creation of an extra virtual disk essential for the {productname} storage partition within VMware environments.
+
+.Procedure: Creating the VMware Virtual Machine
+
+. Download {productname} Server [filename]``.vmdk`` file then transfer a copy to your VMware storage.
+
+. Upload the prepared Ignition or Cloud Init disk file you created using the instructions above.
+
+. Create and name a new virtual machine based on the Guest OS Family [systemitem]``Linux`` and Guest OS Version {sle} 15 (64-bit).
+
+. In [guimenu]``Customize settings`` browse to the uploaded [filename]``.vmdk`` using [systemitem]``IDE controller 0`` storage device in the [systemitem]``Controller Location``.
+
+. Add an additional [systemitem]``Hard Disk 2`` of 500 GB (or more).
+
+. Configure RAM and number of CPUs (at least 16 GB RAM and 4 CPUs).
+
+. Set the network adapter as required.
+
+. Set the [systemitem]``CD/DVD Drive 1`` to use the uploaded configuration as a [systemitem]``Datastore ISO``.
+  Check the box next to btn:[Connect].
+  This drive must be present when you power on the machine initially.
+
+. Power on the VM, and log in using credentials set by configuration disk.
```

Comment:
```suggestion
.Procedure: Creating the VMware Virtual Machine

. Download {productname} Server [filename]``.vmdk`` file then transfer a copy to your VMware storage.

. Make a copy of uploaded [filename]``.vmdk`` using VMWare web interface. This will convert provided vmdk file to the format suitable for vSpere hypervisor.

. Create and name a new virtual machine based on the Guest OS Family [systemitem]``Linux`` and Guest OS Version {sle} 15 (64-bit).

. Add an additional [systemitem]``Hard Disk 2`` of 500 GB (or more).

. Configure RAM and number of CPUs (at least 16 GB RAM and 4 CPUs).

. Set the network adapter as required.

. Power on the VM, and follow firstboot dialogs.
```